pkgs/development/libraries/openjpeg/2.x.nix

```diff
@@ -14,8 +14,12 @@ callPackage ./generic.nix (args // rec {
   patches = [
     ./fix-cmake-config-includedir.patch
     (fetchpatch {
```

Comment:
do you mind adding some comments about what these patches to do.

Reply:
Well I also had to guess, as @TredwellGit added these, but I added a comment.

Reply:
It is to fix finding openjpeg.h in any situation, not just cross-compilation.

Reply:
Thanks, reflected that in the comment.